mlops/bicep/open_sandbox_setup.bicep

```diff
@@ -43,12 +46,6 @@ param siloRegions array = [
 @description('The VM used for creating compute clusters in orchestrator and silos.')
 param computeSKU string = 'Standard_DS13_v2'
 
-@description('The number of nodes used for creating compute clusters in orchestrator.')
-param computeNodes int = 4
```

Comment:
I suppose this deletion was due to a wrong merge. This is fairly minor so let's stick with the magic number - I'll fix it in the next iteration